### DISCUSSION TITLE
Sui Capys

### DISCUSSION THREAD

**Discussion by Ghazwan697** [MystenLabs/sui#8152] [General]
Sui Capys are live on testnet wave 2,  have you tried it ?

**Comment by rm-killer** [comment]
Any one can Say to the dev team that the invite link to the discord doesn't work any more

**Comment by Jensnicolaigustavsen** [comment]
anyone else have been kicked of discord? i cant seem to do pull request and can join back into the discord server either.

**Reply by Kessyboy** [reply]
the bugs on sui capys are too much

**Comment by YungDreezy** [comment]
yeah i did, worked nice!

**Comment by alhaqeeqy93** [comment]
> Sui Capys are live on testnet wave 2, have you tried it ?

Link

**Comment by Ghazwan697** [comment]
Yes, i did and i am doing every day

ma 3. huhtik. 2023 klo 22.29 alhaqeeqy93 ***@***.***)
kirjoitti:
